packages/masonry/README.md

```diff
@@ -25,11 +25,11 @@ pnpm add @solid-primitives/masonry
 
 ## `createMasonry`
 
-Creates a masonry layout calculation from a reactive source array.
+Creates a reactive masonry layout data from a reactive source array.
```

Comment:
Maybe let's not repeat ourselves. Instead of "Creates" again, you could use "Calculates" here.